onnxruntime/python/tools/automl/data_frame_tool_test.py

```diff
@@ -0,0 +1,35 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+
+import argparse
+import onnxruntime as onnxrt
+import numpy as np
+import pandas as pd
+from data_frame_tool import DataFrameTool
+import os
+import sys
+
+def main():
```

Comment:
Is this module using Python's unittest module? If so, it might be good to reframe this test in terms of a unit test so that it can be used with other tooling. 

The code would change to:

```
import unittest

class Suite(unittest.TestCase):
    def test_Tools(self):
        # Existing code here
        self.assertEqual(outputs, [<expected outputs here>])

if __name__ == "__main__":
    sys.exit(unittest.main())
```